docs/src/gettingstarted/next-steps/going-live/first-steps.md

```diff
@@ -20,10 +20,19 @@ Before you take your site live, there are a few steps that will help you prepare
 
 3. **(Optional) Obtain 3rd party SSL if needed**
 
-    Let's Encrypt SSL certificates are automatically issued for Platform.sh projects at no charge to you. If you instead would like to use a [3rd party SSL certificate](/domains/steps/tls.md), make sure that you have purchased it and that it is active prior to going live.
-
-    Additionally, if your application uses [wildcard routes](/domains/steps/tls.md), it will require custom certificates for them. You can manually generate the custom certificates through Let's Encrypt or purchase them separately. We can not automate the generation of these since Let's Encrypt does not support [HTTP challenge](https://letsencrypt.org/docs/challenge-types/) for wildcard certificates.
-
+    Let's Encrypt SSL certificates are automatically issued for Platform.sh projects at no charge to you.
+    
+    The number of Let's Encrypt certificates [is limited](../../../configuration/routes/https.md#limitations).
+    
+    If you want to use a [third-party SSL certificate](../../../domains/steps/tls.md) instead,
+    make sure that you have purchased it and that it's active prior to going live.
+
+     If your application uses [wildcard routes](../../../domains/steps/tls.md), it requires custom certificates for them.
+    
+    You can manually generate the custom certificates through Let's Encrypt or purchase them separately.
+    
+    These can't be generated automatically since Let's Encrypt doesn't support [HTTP challenges](https://letsencrypt.org/docs/challenge-types/) for wildcard certificates.
```

Comment:
I'm not sure each sentence as a separate paragraph is necessary. :shrug: Especially the last three are connected and probably should be part of one paragraph.

In addition, just to align the lines: 
```suggestion
    Let's Encrypt SSL certificates are automatically issued for Platform.sh projects at no charge to you.
    
    The number of Let's Encrypt certificates [is limited](../../../configuration/routes/https.md#limitations).
    
    If you want to use a [third-party SSL certificate](../../../domains/steps/tls.md) instead,
    make sure that you have purchased it and that it's active prior to going live.

    If your application uses [wildcard routes](../../../domains/steps/tls.md), it requires custom certificates for them.
    
    You can manually generate the custom certificates through Let's Encrypt or purchase them separately.
    
    These can't be generated automatically since Let's Encrypt doesn't support [HTTP challenges](https://letsencrypt.org/docs/challenge-types/) for wildcard certificates.
```